decidim-generators/lib/decidim/generators/app_templates/secrets.yml.erb

```diff
@@ -10,6 +10,37 @@
 # Make sure the secrets in this file are kept private
 # if you're sharing your code publicly.
 
+decidim_default: &decidim_default
+  application_name: <%%= (ENV["DECIDIM_APPLICATION_NAME"].presence || "My Application Name").to_json %>
+  mailer_sender: <%%= ENV["DECIDIM_MAILER_SENDER"].presence || "change-me@example.org" %>
+  available_locales: <%%= (ENV["DECIDIM_AVAILABLE_LOCALES"].presence || "en,ca,es").to_s.split(",").map(&:strip).to_json %>
```

Comment:
Not 100% related to this PR, sorry, but I think I'd try to piggyback one thing that have always bothered me... Do we really have a good reason for not adding all the locales from Crowdin that are more than 90% of completion? I mean, if after when you configure your organization you can choose what are the ones that you want to have your platform use. This always looked as an extra configuration step that we could skip. Or is there something for not doing it like that?

Reply:
I don't know, I guess is one of those things that seemed like a good decision at the beginning and the experience is telling us otherwise.
We could also default this variable to `nil` or empty and, in that case, to default it to all the languages available (we should look how the system admin renders all the languages though, it might be a mess).

Reply:
Ok I try this locally with the ones that we have in the crowdin.yaml and indeed it was a mess 😅 . Also, it just doesn't make sense to add languages that aren't translated or maintained, specially for the "`root -> locale -> name`", as that by default is English, so we'll have lots of locales named English. 

So, I think we should curate this list a bit. My proposal rules: 

1. Languages that have [at least 90% of progress in Crowdin ](https://translate.decidim.org/)
2. As we don't differentiate here between a language and its locales, we would have weird things (like "Spanish" 3 times - for :es, :"es-MX" and :"es-PY") so I'd leave only the language and use the locale as a special case. I don't like this as it's a bit colonialist, but I don't have a better solution in mind for now.

With these rules, I have:

```ruby
config.available_locales = [:ca, :cs, :de, :en, :es, :eu, :fi, :fr, :it, :ja, :nl, :pl, :pt, :ro]
```

System panel form screenshot: 

![image](https://user-images.githubusercontent.com/717367/151795898-445eca56-2788-4223-b7b6-ba2d2f13f7b0.png)

Reply:
Perfect! on it!

Reply:
Done in the last commit